style.css

```diff
@@ -0,0 +1,221 @@
+:root {
+  --main-color: #d3ad7f;
+  --black: #13131a;
+  --bg: #fff;
+  --border: 1px solid rgba(255, 255, 0.3);
+  --blueish: #172b4d;
+}
```

Comment:
- [ ] Please change this file name from `style.css` to `index.css` to follow the project requirements.

### See the image of the requirements

![image](https://user-images.githubusercontent.com/83803414/177515540-1c5e47f9-1f2f-4af3-9ad6-474137d2fc5b.png)

Reply:
@thecodechaser 
Thank you 
I missed this . It is fixed now